ci/vendir.yml

```
@@ -1,13 +1,32 @@
 apiVersion: vendir.k14s.io/v1alpha1
 kind: Config
 
+# Relative to ci/
 directories:
-- path: vendor
+- path: ../.github/workflows/vendor
+  contents:
+  - path: . # Copy this folder out to ..
+    git:
+      url: https://github.com/blinkbitcoin/concourse-shared.git
+      ref: kn/mavapay_working
+    includePaths:
+    - shared/actions/*
+    excludePaths:
+    - shared/actions/rust-*
+    - shared/actions/docker-*
+    - shared/actions/chart-*
+    newRootPath: shared/actions
+
+- path: ./vendor
   contents:
   - path: .
     git:
-      url: https://github.com/GaloyMoney/concourse-shared.git
-      ref: 70728a1b62e43be803a337bfae5968b2324f7ea8
+      url: https://github.com/blinkbitcoin/concourse-shared.git
+      ref: kn/mavapay_working
```

to be changed to main
```suggestion
      ref: main
```

---

leave this as it should overwritten by concourse-shared